starknet_py/serialization/factory.py

```diff
@@ -101,7 +102,7 @@ def serializer_for_type(cairo_type: CairoType) -> CairoDataSerializer:
             )
         )
 
-    if isinstance(cairo_type, ArrayType):
+    if isinstance(cairo_type, (ArrayType, FixedSizeArrayType)):
         return ArraySerializer(serializer_for_type(cairo_type.inner_type))
```

Comment:
note: I don't think there's a need to create separate serializer; the aspect of "fixed size" only matters when parsing.